Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -791,6 +791,17 @@ def route_list_values() -> Response:
         elif not values:
             values = "No Results Found For the Query"
 
+        # if the case there are strings to add to the EDL, add them if the output type is text
+        if request_args.out_format == FORMAT_TEXT:
+            append_str = params.get("append_string")
+            prepend_str = params.get("prepend_string")
+            if append_str:
+                append_str = append_str.replace("\\n", "\n")
+                values = f"{values}{append_str}"
+            if prepend_str:
+                prepend_str = prepend_str.replace("\\n", "\n")
+                values = f"{prepend_str}\n{values}"
```

Comment:
Note that the `prepend` has a `\n` between itself and `values`, and the `append` does not have any `\n` or space between itself and `values`. Is that the way it should be?